cloud/linode/cloud.go

```diff
@@ -35,41 +35,14 @@ var Options struct {
 	KubeconfigFlag        *pflag.Flag
 	LinodeGoDebug         bool
 	EnableRouteController bool
+	// deprecated: use VPCNames instead
```

Comment:
Not sure if the deprecation marker is case sensitive: https://go.dev/wiki/Deprecated

Reply:
sounds good, updated